src/core_atmosphere/Registry.xml

```diff
@@ -1447,6 +1455,9 @@
                 <var name="nEdgesOnCell" type="integer" dimensions="nCells" units="-"
                      description="Number of edges forming the boundary of a cell"/>
 
+	        <var name="nCellsOnCellsOnCell" type="integer" dimensions="nCells" units="-"
```

Comment:
Are you sure you want this name? As an independent user, it doesn't make sense to me.

Reply:
This name is from Bill Skamarock. So yes, we want it..